plugin/system_copy.vim

```diff
@@ -71,7 +71,11 @@ function! s:CopyCommandForCurrentOS()
   elseif os == s:windows
     return 'clip'
   elseif os == s:linux
-    return 'xsel --clipboard --input'
+    if !empty($WAYLAND_DISPLAY)
+      return 'wl-copy'
```

Comment:
I guess something else I could also do is detect if this is installed (its not always installed by default). However the alternative `xsel` would not work under Wayland anyway.